common/src/main/java/org/apache/nemo/common/ir/vertex/executionproperty/MinParallelismProperty.java

```diff
@@ -21,15 +21,20 @@
 import org.apache.nemo.common.ir.executionproperty.VertexExecutionProperty;
 
 /**
- * Parallelism ExecutionProperty.
+ * This property decides the minimum number of parallel tasks to use for executing the corresponding IRVertex.
+ *
+ * The actual number of tasks used in the execution can be greater due the following reasons.
+ * - A larger number of parallelism of a parent IRVertex connected with an one-to-one IREdge.
```

Comment:
Is this possible? doesn't it check whether or not it has the same parallelism before connecting with a 1-1 edge?